### PR TITLE
Update syn, quote, proc_macro2

### PR DESCRIPTION
**Pull request by Eijebong**

### CODE REVIEW

**Comment by Eijebong** [comment]
Ping ? :)

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "derive_more"
-version = "0.11.1-pre"
+version = "0.11.1"
```
Comment (JelteF) [inline Cargo.toml]: Don't change the version yet, I will do that when releasing.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## 0.11.1 - 2018-07-30
```
Comment (JelteF) [inline CHANGELOG.md]: Keep this as Unreleased for now

**Comment by Eijebong** [comment]
Not sure you get notifications when I push on the PR so... Done :p

**Comment by JelteF** [comment]
Looks good, I'll try to create a release in the near future.

**Comment by JelteF** [comment]
I just released version 0.12.0, sorry this took so long.

**Pull request merged**